transport/mixer/src/data.rs

```diff
@@ -0,0 +1,39 @@
+use std::cmp::Ordering;
+
+/// Data structure holding the data alongside a release timemestamp.
+///
+/// The ordering functionality is defined only over the release timestamp
+/// to ensure proper mixing.
+pub struct DelayedData<T> {
+    pub release_at: std::time::SystemTime,
```

Comment:
Please use `std::time::Instant`, since you don't seem to need to know the actual clock.